### PR TITLE
Testing mkay

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

File: templates/upstart.conf.j2 [hunk 1]
```diff
@@ -21,7 +21,7 @@ script
     docker rm -f {{ modcloth_app_name }} || true
   fi
 
-  docker run --sig-proxy --rm \
+  docker run \
     -p {{ modcloth_app_docker_port }}:{{ modcloth_app_docker_port }} \
```
Comment (rafecolton) [inline templates/upstart.conf.j2:25]: These two should be allowed to be different.

Also, what if I want to `--link` another container or use `--volumes-from`?

Comment (meatballhat) [inline templates/upstart.conf.j2:25]: That's what I made it so that you can override the template name.

Comment (rafecolton) [inline templates/upstart.conf.j2:25]: ahhh :thumbsup: :thumbsup:

**Comment by rafecolton** [comment]
Good to merge?

**Comment by jszwedko** [comment]
:shipit:

**Comment by meatballhat** [comment]
@rafecolton think so yup :)

**Pull request merged**